### DISCUSSION TITLE
Explain the "Special partners" on onyx.schwarz

### DISCUSSION THREAD

**Discussion by patrick246** [SchwarzIT/onyx#348] [Ideas]
The special partners section of the landing page does not explain them further. STACKIT is linked, which enables visitors to learn more about it. Wawi Nexus and Core UI are not explained or linked, so visitors have no idea what they mean.

The description texts could include Wawi Nexus adopting Onyx as the ui framework, or the team building Onyx being able to draw from the experiences building and maintaining CoreUI, as well as explaining what these even are.

**Comment by larsrickert** [comment]
We will talk about this in #699 :)
